golos.publication/golos.publication.cpp

```diff
@@ -787,4 +791,35 @@ void publication::reblog(name rebloger, name author, std::string permlink) {
             "You can't reblog, because this message doesn't exist.");
 }
 
+int64_t publication::pay_delegators(uint64_t message_id, int64_t claim, name voter, 
+        eosio::symbol tokensymbol, std::vector<structures::delegate_voter> delegate_list) {
+    int64_t dlg_payout_sum = 0;
+    for (auto delegate_obj : delegate_list) {
+        if (delegate_obj.payout_strategy == config::to_delegated_vesting) {
+
+            auto interest_rate = static_cast<uint64_t>(static_cast<uint128_t>(delegate_obj.quantity.amount) * 
+                    delegate_obj.interest_rate / delegate_obj.effective_vesting.amount);
+
+            auto dlg_payout = claim * interest_rate / config::_100percent;
+
+            tables::account_table acc_table(_self, voter.value);
```

Comment:
You try to modify table in another smart-contract. You can't do this. For modify data in another smart-contract you should call his actions.